config/CLIConfiguration.ts

```diff
@@ -257,13 +257,29 @@ class _CLIConfiguration {
 
   getConfigForAccount(accountId?: number): CLIAccount_NEW | null {
     if (this.config) {
-      this.config.accounts.find(account => account.accountId === accountId) ||
-        null;
+      return (
+        this.config.accounts.find(account => account.accountId === accountId) ||
+        null
+      );
+    }
+    return null;
+  }
+
+  getConfigAccounts(): Array<CLIAccount_NEW> | null {
+    if (this.config) {
+      return this.config.accounts || null;
```

Comment:
mega nit: for this and `getConfigForAccount` above you can use the `?.` operator to avoid having to do the `if` statement. There are maybe a few other places where this is applicable

Instead of
```
if (this.config) {
      return this.config.accounts || null;
}
return null
```
it would be
```
return this.config?.accounts || null
```